stdlib/ostruct.rb

```diff
@@ -60,10 +66,52 @@ def to_n
   def hash
     @table.hash
   end
+  
+  def table
```

Comment:
If it's a JS object, access will likely be faster, but if you need `Hash` methods feel free to use one.